src/olympia/addons/views.py

```diff
@@ -187,13 +187,18 @@ def find_replacement_addon(request):
 class AddonViewSet(
     CreateModelMixin, RetrieveModelMixin, UpdateModelMixin, GenericViewSet
 ):
+    _write_permissions = [
```

Comment:
Why the leading underscore in this variable name? How is it different from, for example, `permission_classes` below?

Reply:
it was supposed to be a private class property (well, not really, because python doesn't have those except by convention) but it ended up being referenced in other classes so I guess that isn't true any longer.  
`permission_classes` is a property on `GenericViewSet` we're overriding/setting btw.

Reply:
renamed in https://github.com/mozilla/addons-server/pull/18383/commits/777df1f9cd22b2b061c4e2675b2327f4c7f95b99